companies/costco.md

```diff
@@ -0,0 +1,10 @@
+---
+forbes_rank: 142
```

Comment:
```suggestion
forbes_rank: 14
```

It looks like Costco is #14 .  I need to update this from forbes_rank to fortune_rank 🤦🏻  Sorry about that
https://fortune.com/company/costco/fortune500/